docs/try_local_system.md

```diff
@@ -476,10 +478,12 @@ Once you're done with setting up the dependencies, proceed with
    cd hyperswitch
    ```
 
-3. Run database migrations using `diesel_cli`:
+3. Run database migrations:
+
+   Note that this requires `diesel_cli` to be installed
 
    ```shell
-   diesel migration --database-url postgres://$DB_USER:$DB_PASS@localhost:5432/$DB_NAME run
+   make migrate database-url=postgres://$DB_USER:$DB_PASS@localhost:5432/$DB_NAME
    ```
```

Comment:
We need not mention that they'd need `diesel_cli`, since our setup guides already include steps for installing it. However, we'd have to update our guides to include `make`, just in case.

- Update the Ubuntu and Ubuntu on WSL2 guides to install the `make` package using `apt`. (You can edit the existing commands which install `pkg-config` and other packages.
- I _think_ recent versions of MacOS have `make` installed by default.
- For native Windows, we'll have to include a command like so: `winget install ezwinports.make`. (Although from what I remember, we'd have to revisit native Windows setup entirely.)

Reply:
> Update the Ubuntu and Ubuntu on WSL2 guides to install the make package using apt

The build essential package already includes make, we are installing this on wsl. Have made the changes to ubuntu setup guide to install make along with openssl.